### PR TITLE
service.py: Added new method statuslog() to show the service status text

### PR DESCRIPTION
**Pull request by svirt**
New method statuslog() is added. It is needed for getting the status log
text of the service. The existing status() method returns only the status
if the service runs or not

### CODE REVIEW

**Comment by yangdongsheng** [comment]
Is there a use case for this function? It would be great mentioning it here to a reviewer. Thanx

**Comment by svirt** [comment]
Yes, it is used in tests in virsh_managedsave.py, but I still didn't commit the modified tests yet because they depend on this pull request result

File: virttest/staging/service.py [hunk 1]
```diff
@@ -129,6 +129,15 @@ def method(cmdResult):
...
             return bool(re.search(r"running", output))
         return method
+    elif command == "statuslog":
+        def method(cmdResult):
+            """
+            Parse method for service XXX status.
+
+            Returns status log
+            """
+            return cmdResult
```
Comment (yangdongsheng) [inline virttest/staging/service.py]: But from the commit message and the name of "statuslog", I am expecting to get a string when I call service.statuslog(), while it's a CmdResult object currently. I am not sure which one is better in your usecase. But I have two suggestion here.

1, change the statuslog to rawstatus or something else, and reword the comments for it. Keep returning a CmdResult object.
2. return cmdresult.output in parser. Keep the comment and commit message.

Either way is acceptable to me, I think. What do you think about it?

Yang

**Comment by svirt** [comment]
After merging this pull request I will be able to commit the use case test

**Comment by yangdongsheng** [comment]
@svirt
Firstly, please address my comment inline, I have shown my concern and my suggestions in that comment.

Secondly, please feel free to send a PR which is depending on this patch. It's better to understand your usecase and to review it.

Thanx.

**Comment by svirt** [comment]
**@yangdongsheng**
Hello, 
yes, I think your mentioned first case is more suitable -  and the name rawstatus looks better, I can rename and rewrite the comments for it.

If it's fine for you, I will do the necessary changes.

Thanks

**Comment by yangdongsheng** [comment]
@svirt
Great!

Please rename it to `raw_status`. others look good to me in this PR.

Thanx

**Comment by svirt** [comment]
**@yangdongsheng**
Yes, I will create a new PR with these changes. Thanks

**Comment by yangdongsheng** [comment]
No need at all. Just update the branch in your local repo and push it to github. Then this PULL REQUEST will be updated automatically.

**Comment by svirt** [comment]
Ok. I will update this branch

**Comment by svirt** [comment]
Looks like there is a problem with running Travis CI on github's side, the log shows:

**fatal: unable to connect to github.com:**
**github.com[0: 192.30.252.129]: errno=Connection timed out**

more details are in the log

**Comment by svirt** [comment]
closed/reopened the PR  to run Travis CI again...

**Comment by waynesun09** [comment]
@svirt could you squash your two commits into one, thx

**Comment by svirt** [comment]
**@waynesun09**
yes sure,done. Now there is one commit only

**Comment by waynesun09** [comment]
ACK
@svirt @yangdongsheng Thanks! merging now.

File: virttest/staging/service.py [hunk 2]
```diff
@@ -270,6 +289,9 @@ def sys_v_init_command_generator(command):
     elif command == 'disable':
         command_name = "chkconfig"
         command = "off"
+    elif command == "raw_status":
+        command_name = "chkconfig"
+        command = "status"
```
Comment (yangdongsheng) [inline virttest/staging/service.py]: Why the command_name is chkconfig?

Comment (yangdongsheng) [inline virttest/staging/service.py]: I guess you did not test this patch on a sysV environment yet. I think the command_name should be service in here. Do you mind send another patch for it?

**Comment by svirt** [comment]
**@yangdongsheng**
Hi, yes I agree, the command_name = "service" is defined above already and command_name = "chkconfig" is not needed .
I will remove that line. Shell I request a new pull request?

**Comment by yangdongsheng** [comment]
Yes, as there is no need to rename the command_name and the command, I think this branch is whole not needed. It's better to do a test for it I suppose on a sys_v style environment. Thanx for your work.

**Comment by svirt** [comment]
**@yangdongsheng**
Thanks for the review, I will do some checks and then prepare another pull request
